tests/framework/session/test_session_hook_manager.py

```diff
@@ -58,19 +53,19 @@ def test_calling_register_hooks_twice(self, project_hooks):
     def test_hooks_registered_when_session_created(
```

Comment:
I love that we already had a test with this name. Makes this change feel all the more natural. 💪